### DISCUSSION TITLE
Github Student Dev Pack

### DISCUSSION THREAD

**Discussion by PythonAtSea** [community/community#58279] [GitHub Education] [answered]
### Hi everyone,

I'm a homeschool student in Mississippi. Because of this I don't have a student id or a school email. However, I believe I should qualify for the pack. How should I go about applying for the pack?

--Thanks in advance, pythonatsea

### A suggestion...(optional)

Have better support for homeschoolers applying for the GH Student Dev Pack.

**Comment by the-officialjosh** [comment]
If you are a homeschool student in Mississippi and do not have a student ID or a school email, you can still apply for the GitHub Student Developer Pack. While the pack is primarily designed for students enrolled in accredited educational institutions, GitHub also considers applications from homeschooled students. It says so at the page itself

To apply for the pack, you can follow these steps:

1. Visit the GitHub Education website at [education.github.com](https://education.github.com).
2. Scroll down and click on the "Get the Pack" button.
3. On the next page, click on the "Get benefits as a student" button.
4. You will be prompted to authenticate with your GitHub account. If you don't have one, you can sign up for a free account.
5. After logging in, you will see an application form. Fill in the required information, including your personal details and educational background.
6. In the field where it asks for your school name and email, you can provide information related to your homeschooling setup. You can mention that you are a homeschooled student in Mississippi and explain your situation briefly.
7. Submit the application form.

GitHub will review your application, and if they require any additional information or verification, they will reach out to you via email. It's important to provide accurate and honest information in your application.

Remember, while there is no guarantee of acceptance, GitHub evaluates applications on a case-by-case basis, and being a homeschooled student should not automatically disqualify you from the pack.

Good luck with your application, and if you have any further questions, feel free to ask!

**Reply by PythonAtSea** [reply]
Thanks for your reply. I’m not quite sure what you want me to do. If I select my school as homeschool then I get a thing saying take a picture of your student ID. I don’t have a student ID.
![IMG_0852](https://github.com/community/community/assets/61562046/df77e69e-c79e-4e18-ac4a-9053bc254c1d)

Do you have any idea what I should do? 

—Thanks in advance, pythonatsea

**Reply by the-officialjosh** [reply]
I did not go that far while trying to get help there for my reply but you could try this -- For the email field, provide your personal email address instead of a school email.
In the field where it asks for a student ID, you can explain that as a homeschooled student, you don't have a student ID.
From what I see the the top says of academic status and not student ID. What is in the dropdown is there anything there you have at least
Complete the remaining sections of the application form with accurate information.
Submit the application form.

**Reply by PythonAtSea** [reply]
What should I upload as the picture?

**Reply by the-officialjosh** [reply]
I asked for the content of the dropdown that will help me be guide you on what to put there

**Reply by PythonAtSea** [reply]
Here's the contents of the drop-down. I might still have a score sheet from a math competition from spring of this year and maybe that would work.

![image](https://github.com/community/community/assets/61562046/cb576bb0-fa88-4917-867e-71de3d11007f)

**Reply by the-officialjosh** [reply]
Among these is there anything that you have that can fill in for any, because it seems that question is compulsory? If there is any evidence that you have of your schooling that is not listed here you could go with the other option and the submit. I hope you have an evidence of your being homeschooled because they just need something to prove your status

**Reply by PythonAtSea** [reply]
I don't know. In Mississippi to homeschool all you have to do is fill out a very short form and send it to your local school attendance officer. They don't give you any kind of paperwork that says that you're homeschooled. It's kind of annoying. [Mississippi DOE page](https://www.mdek12.org/OCSA/HS)

**Reply by the-officialjosh** [reply, marked as answer]
I understand that not having evidence of being homeschooled can make it challenging to meet the requirements. One option you could explore is reaching out to your local government or education authorities to inquire if they can provide you with a letter or any form of verification that confirms your homeschooling status. These measures were implemented to ensure that the student developer pack is accessed by individuals who genuinely meet the eligibility criteria. It's worth exploring the possibility of obtaining official documentation to support your application.

**Reply by PythonAtSea** [reply]
Thanks for your help!

**Reply by the-officialjosh** [reply]
You're welcome

**Reply by Dilesha27** [reply]
Hi @the-officialjosh ,
         I already have student developer pack. but, I got this mail. I attached that here. would you know how to apply student developer pack again? 
![Screenshot (23)](https://github.com/community/community/assets/88300193/086190f6-7f7d-49c9-9140-2c62dfa0bf7f)

**Comment by C0deM1ner** [comment]
hello everyone i have a problem with the student package.. after submitting a request i get a message in my gmail with my old name but i have updated that

**Reply by the-officialjosh** [reply]
Check the time of the mail, my younger bro had similar issues but by looking at the time stamp you will notice that you were mailed at that first instance before updating your name

**Comment by saleh-2023** [comment]
hello everyone, 
 I am writing to seek assistance with an issue I encountered while trying to access Copilot Free for my student account.
My student email address, saber.saleh@graduate.utm.my, has been used to obtain an academic discount on GitHub account(s) under the username Zkri-Saber. However, despite meeting the eligibility criteria, I am currently unable to utilize Copilot Free.
I would greatly appreciate it if you could help me resolve this matter so that I can benefit from the Copilot feature as a student. I have been eagerly looking forward to using this powerful tool to enhance my coding experience and improve my productivity.
Please let me know what steps I need to take in order to enable Copilot Free on my student account. If there are any additional documents or information required to verify my eligibility, please inform me, and I will gladly provide them.
Thank you for your attention to this matter. I sincerely appreciate your assistance in resolving this issue promptly. I look forward to your positive response and the opportunity to utilize Copilot Free as a student.

Kind regards,
![image](https://github.com/community/community/assets/132210412/b9191a8d-024c-4058-a9a4-7b6b289a0a5b)

**Reply by the-officialjosh** [reply]
Simply put you can't use on email especially this student email to get a discount on one and then get the pack on another.

**Comment by Marig-collab** [comment]
Good morning, I am Head of Career at Le Wagon and we have your student dev pack as a benefit for our student. 

Nevertheless for the last 6 months, students got often a lot of difficulties to connect to it. We are using this link : https://education.github.com/student/verify?school_id=22492&student_id=kitt_17436&signature=3b6bbcd54d180749eab384d4ca2f88afc0a8751a50f95e2aca33bfb838cd09fc

Could you help me please?
Thanks

**Reply by PythonAtSea** [reply]
Just looking at the link it looks like a one time code.

**Reply by Marig-collab** [reply]
Ok so what link should I use to be share with my students please?

**Reply by the-officialjosh** [reply]
This is what I see when  I log in ```Oops! Looks like this code was already redeemed.
If you believe this is an error please contact your administrator.``` It barely explains what is going on. can you explain better so I might understand and try to help you?

**Reply by fareen87** [reply]
Hi @the-officialjosh. I hope you're well. I work with Marig in the Careers team at LeWagon. I wanted to follow up on this thread - would you be able to help us with the right URL we can share with our students and alumni? 

At the moment this is the link we have on our internal perks page for students and alumni to avail and seems like it's not valid. It would be great if you can help us with a valid link that we can share with everyone.

Is there any other information you need to move forward?

**Comment by Metro1998** [comment]
Hello, i have gotten approved, but i didnt get the email and student dev pack, is it normal？

**Reply by the-officialjosh** [reply]
Typically, once your application is approved, you should receive an email notification confirming your approval. In most cases, the Pro status is one of the first features to be rolled out to you. It's important to wait for a while and keep an eye on your email inbox. 

Regarding your question about knowing if you were approved, it's important to note that the approval process does not work immediately after submission. The GitHub Education team will review the applications and notify you via email once your application has been approved. Therefore, receiving an email confirmation is typically how you will know if your application has been approved.

Please be patient and keep an eye on your email for any updates or notifications from the GitHub Education team. If you have already submitted your application, it's a matter of waiting for the approval process to be completed.

**Reply by Metro1998** [reply]
Got it! thank you for your reply.

**Comment by ShoaibArshad10** [comment]
Hello everyone! I'm looking for guidance on how to enroll in ALGOEXPERT.IO and take advantage of their amazing "Student dev pack offers." If any of you have experience with this, I would greatly appreciate your assistance. Could you please share the steps I need to follow to sign up for ALGOEXPERT.IO and access the student discounts? Your guidance will help me make the most of this opportunity and enhance my learning as a student. Thank you in advance for your help!

**Comment by amanmunjekar** [comment]
Subject: Request to Reinstate Access to GitHub Student Developer Pack

Dear GitHub Support,

. My name is Aman Munjekar, and I am writing to request the reinstatement of my access to the GitHub Student Developer Pack.

I am a current user of GitHub and have previously been approved for the GitHub Student Developer Pack [6/5/22 of myr previous approval]. However, due to an accidental resubmission of my application, my access to the student developer pack has been revoked.

I apologize for any inconvenience caused by this mistake.

I would greatly appreciate it if you could reinstate my access to the GitHub Student Developer Pack.  

Please let me know if there are any additional steps I need to take or any further information you require to resolve this matter. I am more than willing to provide any necessary details.

Thank you for your attention to this matter. I look forward to your prompt response and the opportunity to regain access to the GitHub Student Developer Pack.

Best regards,

Aman Munjekar.
username: amanmunjekar.

**Comment by amanmunjekar** [comment]
Subject: Request to Reinstate Access to GitHub Student Developer Pack

Dear GitHub Support,

. My name is Aman Munjekar, and I am writing to request the reinstatement of my access to the GitHub Student Developer Pack.

I am a current user of GitHub and have previously been approved for the GitHub Student Developer Pack [6/5/22 of myr previous approval]. However, due to an accidental resubmission of my application, my access to the student developer pack has been revoked.

I apologize for any inconvenience caused by this mistake.

I would greatly appreciate it if you could reinstate my access to the GitHub Student Developer Pack.

Please let me know if there are any additional steps I need to take or any further information you require to resolve this matter. I am more than willing to provide any necessary details.

Thank you for your attention to this matter. I look forward to your prompt response and the opportunity to regain access to the GitHub Student Developer Pack.

Best regards,

Aman Munjekar.
username: amanmunjekar.

**Comment by amanmunjekar** [comment]
Use admission card instead of college id ...

**Comment by yetkinkrsngr** [comment]
hey i have a verifity univercity email adress but still can not be student account. i has been send turksih higher education institution student letter it is rejected and triyng to send my transcript and my student bus card i cannot do that giving me error what should i do ?

**Comment by amanmunjekar** [comment]
Try admission card

**Reply by yetkinkrsngr** [reply]
i dont have it Distance/Online Learning

**Comment by amanmunjekar** [comment]
Use admission card, estimation card try every card which is related to your
school until you don't get approved..... I also tried like that...

**Comment by urnotop** [comment]
hello my name is Phinehas, i am studying in 7th class, 11 years old. I live in Hyderabad. I am homeschooled by ACE. this is their website - https://www.aceconnect.com/. How do I apply for the student educational pack
